File: sdkwork-backend-sdk-python/sdkwork_backend_sdk/models/plus_ai_tool_vo.py
from dataclasses import dataclass
from typing import Optional, List, Dict, Any

@dataclass
class PlusAiToolVO:
    """AI工具VO类"""
    created_at: str = None
    updated_at: str = None
    id: int = None
    name: str = None
    description: str = None
    type: str = None
    status: str = None
    enabled: bool = None
    http_api_definition: HttpApiDefinition = None
    tool_definition: PlusToolDefinition = None

File: sdkwork-backend-sdk-go/types/plus_ai_tool_vo.go
package types

// AI工具VO类
type PlusAiToolVO struct {
	CreatedAt string `json:"createdAt"`
	UpdatedAt string `json:"updatedAt"`
	Id int `json:"id"`
	Name string `json:"name"`
	Description string `json:"description"`
	Type string `json:"type"`
	Status string `json:"status"`
	Enabled bool `json:"enabled"`
	HttpApiDefinition HttpApiDefinition `json:"httpApiDefinition"`
	ToolDefinition PlusToolDefinition `json:"toolDefinition"`
}
